### ISSUE TITLE
Error: Second attempt: material-components-web.min.js:38 Uncaught TypeError: Cannot read property 'querySelector' of null at e.value (material-components-web.min.js:38) at e.t (material-components-web.min.js:6) at new e (material-components-web.min.js:38) at index.html:146 

### ISSUE THREAD

**Issue by jeffsmithsailing** [material-components/material-components-web#1620]
Resubmitting - 1618 closed inappropriately. The problem with the CodePen is actually the reason I raised the error - the sample code on the MDC-Web site is not working.

What MDC-Web Version are you using?

"latest" 0.25.0 via <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>

Please be specific, e.g. major.minor.patch
What browser(s) is this bug affecting?

Doing my debugging in Chrome. Will test in others when I get this issue fixed.

Please include the browser version. A user-agent string is also quite helpful.
Version 61.0.3163.100 (Official Build) (64-bit)
What OS are you using?

Windows 10

Please include the OS version.
10.0.15063 Build 15063
What are the steps to reproduce the bug?

I am attempting to create a prototype of the MDC-Web tab control using the code at https://material.io/components/web/catalog/tabs/#script. Specifically, I am trying to create a tab bar with icons and text labels.

image

The page loads and the tabs are presented, but only the active tab is enabled. The other tabs do not respond. I will create a CodePen illustrating the issue.

image

At http://material-components-web.appspot.com/tabs.html#nearby I found that the syntax for the JS src is <script src="/assets/material-components-web.js"></script> in contrast to <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script> which is in the demo site. The sample for Icon & Text Labels works perfectly although the sample immediately below [CSS-only icon & text labels] is not working and is displaying the same behavior as my prototype.
image

I believe the problem is related to the new version of the CSS (0.25.0) released last week.

Here is the CodePen illustrating the problem:
https://codepen.io/jeffreysmithixd/pen/yPvKmP
I moved the Autoinit from the JS pane to the HTML pane. The remainder of the JS is from the MDC-Web site called out above. It is not working.

What is the expected behavior?

The sample code on the demo site works and I can create a functioning prototype of the tab control.

Please describe what the component/code should be doing that it's not.
The active tab is in focus and the other tabs cannot be selected.
Any other information you believe would be useful?

Hopefully this is not a duplicate - I searched but found older instances of issues with the querySelector error that were closed.

**Comment by kfranqueiro** [comment]
Your CodePen seems to have a lot of script tags mixed into the JS pane, which would contribute to it failing to load.

Firstly, to address some confusion, the `/assets/...` URLs in the deployed demo pages point to versions built locally as a direct result of running `npm run dev` in a local copy of the MDC Web github repository. These are normal to see in the demo pages. The assumption is that you will either reference your own local MDC Web installation as described in the main README, or use unpkg as described in the getting started guide.

Secondly, I would point out that the JS code you pasted from the demo does not match your own markup. The demo JS instantiates `MDCTabBar` on an element with the ID `dynamic-tab-bar`, which doesn't exist in your markup.

Thirdly, your example's markup also doesn't contain the `dots` and `dot` elements that are in the demo code you pasted. Removing references to those further eliminates errors.

I've assembled the beginnings of a [functioning CodePen](https://codepen.io/kfranqueiro/pen/MOQPaR) based on our [template](https://codepen.io/traviskaufman/pen/pNQmRp) which already loads the latest version from unpkg.

I've gone as far as addressing the issues I pointed out above, but I believe there may still be issues with your markup and/or the JS to properly synchronize the active panel and organize the contents of each panel as desired. Hopefully this is enough to get you on the right track, though.
